src/style.css

```diff
@@ -1,3 +1,63 @@
-body {
+* {
+  box-sizing: border-box;
   margin: 0;
+  padding: 0;
+}
+
+:root {
+  --blue: #00acdc;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+}
+
+.logo {
+  padding: 10px 0;
+  cursor: pointer;
+  height: 40px;
+  width: 40px;
+}
+
+.nav__list {
+  list-style: none;
+}
+
+.nav__item {
+  display: inline-block;
+  margin: 0 20px 0 0;
+}
+
+.nav__item:last-child {
+  margin: 0;
+}
+
+.nav__link {
+  position: relative;
+  display: inline-block;
+  font-family: Roboto , sans-serif;
+  font-weight: 500;
+  text-transform: uppercase;
+  text-decoration: none;
+  font-size: 12px;
+  color: #000;
+  transition: 0.2s;
+}
+
+.nav__link:hover {
+  color: var(--blue);
+}
+
+.is-active::after {
+  content: "";
+  position: absolute;
+  background-color: var(--blue);
+  height: 4px;
+  width: 100%;
+  border-radius: 8px;
+  bottom: -21px;
```

Comment:
must be bottom: 0
<img width="596" alt="Screenshot 2023-08-31 at 17 18 05" src="https://github.com/mate-academy/layout_moyo-header/assets/53228259/19950ade-30c8-4c86-99e5-804136e2a2fb">
<img width="1324" alt="Screenshot 2023-08-31 at 17 18 15" src="https://github.com/mate-academy/layout_moyo-header/assets/53228259/6078ed58-d61b-41b0-bb95-fde52f1e6c4c">

and your elements should be full width also the element with is-active class should be #00acdc